### PR TITLE
Fix typos in cli_tutorial.md

### PR DESCRIPTION
**Pull request by IAlibay**
Currently up to analysis step

### CODE REVIEW

**Comment by IAlibay** [comment]
That's all my comments for the CLI portion, I'll look at the notebook next.

File: rbfe_tutorial/cli_tutorial.md [hunk 1]
```diff
@@ -75,25 +76,24 @@ openfe view-ligand-network network_setup/ligand_network.graphml
...
+   20-second timeout, core-core element changes disallowed, and max3d set to 1.
 2. The network is a minimal spanning tree, with the default LOMAP score used to
    score the mappings.
-3. Solvent is water with NaCl at an ionic strength of 0.15 M (neutralized).
-4. The protocol used is OpenFE's OpenMM-based RFE protocol, with default settings.
-
-<!-- TODO there should be a link to the default settings here -->
+3. Solvent is water with NaCl at an ionic strength of 0.15 M (neutralized) with a
+   minimum distance of 1.2 nm from the solute to the edge of the box.
+5. The protocol used is OpenFE's OpenMM-based Hybrid Topology RFE protocol, with [default settings](https://docs.openfree.energy/en/stable/reference/api/openmm_rfe.html#protocol-settings).
```
Comment (dwhswenson) [inline rbfe_tutorial/cli_tutorial.md:96]:
```suggestion
4. The protocol used is OpenFE's OpenMM-based Hybrid Topology RFE protocol, with [default settings](https://docs.openfree.energy/en/stable/reference/api/openmm_rfe.html#protocol-settings).
```

yeah, markdown will catch it, but still

File: rbfe_tutorial/cli_tutorial.md [hunk 2]
```diff
@@ -192,9 +192,8 @@ This will write out a tab-separated table of results, including both the
 $\Delta G$ for each leg and the $\Delta\Delta G$ computed from pairs of legs.
 The first column is a description of the data, e.g., `DGcomplex(ligandB,
 ligandA)` for the $\Delta G$ of the transformation of ligand
-A into ligand B in vacuum, or `DDGbind(ligeandB, ligandA)` for the
-$\Delta\Delta G$ of binding ligand A vs. ligand B: $\Delta G$<sub>bind,
-$B$</sub>$ - \Delta G$<sub>bind$A$</sub>. The second column tells the type of
+A into ligand B complexed to a protein, or `DDGbind(ligeandB, ligandA)` for the binding
```
Comment (dwhswenson) [inline rbfe_tutorial/cli_tutorial.md:195]:
```suggestion
A into ligand B complexed to a protein, or `DDGbind(ligandB, ligandA)` for the binding
```

Suggesting a fix to my typo in your PR

**Review by dwhswenson — APPROVED** [review]
Minor fixes to the fixes.

**Comment by dwhswenson** [comment]
This... probably should have been merged BEFORE the workshop? 🤷‍♂️ Merging now.

**Pull request merged**